src/tag.zig

```diff
@@ -34,6 +34,10 @@ pub const Tag = enum(u32) {
 		loadDefaults();
 	}
 
+	pub fn get(tag: []const u8) ?Tag {
```

Comment:
Maybe we should name this one `find`, and the other one `findOrInsert`, but I'll leave that for a future PR.

Reply:
There is already similar construct in HashMap - `get` and `getOrPut`